src/pages/index.tsx

```diff
@@ -94,49 +97,50 @@ export default function Home() {
         />
 
         <h2 className="text-center text-[52px] font-bold text-red">
-          Begin Your Journey
+          Beginning Your Journey
         </h2>
         <VectorHorizontal className="mx-auto" />
         <div className="my-6 -ml-36 flex w-[494px] flex-col gap-6 rounded-[32px] border border-white px-11 pb-14 pt-24 text-center text-black">
-          <h3 className="text-[32px] font-bold">
-            Embark on the RetroPGF Odyssey
-          </h3>
+          <h3 className="text-[32px] font-bold">Simplifying Project Review</h3>
           <p className="text-lg font-medium">
-            Dive into the RetroPGF universe and discover a realm of projects
-            that hold the promise of positive change. Browse through our
-            collection and get a sneak peek into the potential impact you can
-            drive.
+            Crafted for RPGF, Pairwise transforms the overwhelming task of
+            reviewing numerous projects into a manageable and focused activity.
+            Concentrate on the projects you know best, and contribute to a more
+            informed and balanced allocation of funds.
           </p>
         </div>
         <VectorHorizontal2 className="mx-auto" />
         <div className="my-6 -mr-32 ml-auto flex w-[494px] flex-col gap-6 rounded-[32px] border border-white px-11 pb-14 pt-24 text-center text-black">
-          <h3 className="text-[32px] font-bold">Galactic Governance Awaits</h3>
+          <h3 className="text-[32px] font-bold">Streamlining List Creation</h3>
           <p className="text-lg font-medium">
-            Step into the RetroPGF cosmos, where your votes will determine the
-            trajectory of projects destined to shape our universe. Each
-            selection you make sends ripples through the space of innovation.
+            With Pairwise, you can easily create, edit, and share curated lists
+            of projects, to be imported directly into Agora and Supermodular for
+            final voting. By progressing through planets and moons, the process
+            of list-making is engaging and fun!
           </p>
           <CollectionsLeft className="absolute -left-[350px] -mt-[300px]" />
         </div>
         <VectorHorizontal3 className="mx-auto" />
         <div className="my-6 -ml-36 flex w-[494px] flex-col gap-6 rounded-[32px] border border-white px-11 pb-14 pt-24 text-center text-black">
-          <h3 className="text-[32px] font-bold">Unlock More Collections</h3>
+          <h3 className="text-[32px] font-bold">Leveraging Your Expertise</h3>
           <p className="text-lg font-medium">
-            Unleash your influence by voting on the unlocked collection. Your
-            votes are your voice, shaping the future of these projects. Every
-            click is a step toward making a difference.
+            Pairwise streamlines the review process, guiding you to the
```

Comment:
@mosaeedi 

Graphics still uses wrong terms. `Begin ranking` should be used, not `Awaiting direction`.

<img width="639" alt="image" src="https://github.com/GeneralMagicio/pw-frontend/assets/9698363/b745e8ad-bce9-4154-b709-a8f367ef1634">

Reply:
I've uploaded the updated version with the correct copy!
I'll review it again.

Reply:
Creted a separate issue for this #121